src/visibility.js

```diff
@@ -154,8 +154,8 @@ class Visibility extends BasePlugin {
     this.eventManager.listen(this.player, this.player.Event.LEAVE_PICTURE_IN_PICTURE, () => {
       this._isInPIP = false;
     });
-    this.eventManager.listen(this.player, this.player.Event.FIRST_PLAYING, () => {
-      this._everStartedPlaying = true;
+    this.eventManager.listen(this.player, this.player.Event.PLAYBACK_START, () => {
```

Comment:
Are you looking for interaction with the player? if so I think `USER_GESTURE` event gives you the indication otherwise keep it like that.

Reply:
user gesture means user interacted with the player, and would not take into account autoplay, so playback start is the one we want

Reply:
OK